bundlemeta.json

```diff
@@ -0,0 +1 @@
+{"chunks":{"ui.es.js":{"id":"ui.es.js","isEntry":true,"size":817,"sizeGzip":352,"sizeBrotli":291,"contents":{}},"ui.cjs.js":{"id":"ui.cjs.js","isEntry":true,"size":798,"sizeGzip":357,"sizeBrotli":295,"contents":{}}}}
```

Comment:
Should this be committed or ignored?

Reply:
We don't need to keep it. I'm adding it to `.gitignore`. 

I've set the build to open the bundle size UI on the browser. This is what it looks like in the browser:

![image](https://github.com/user-attachments/assets/6fc2b764-4e98-490d-9b42-a3dd80866273)

Reply:
Fixed